airflow/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -76,7 +88,9 @@ def execute_query(
         wait_for_completion: bool = True,
         poll_interval: int = 10,
         workgroup_name: str | None = None,
-    ) -> str:
+        session_id: str | None = None,
+        session_keep_alive_seconds: int | None = None,
+    ) -> QueryExecutionOutput:
```

Comment:
This is a breaking change. Users who are using `RedshiftDataHook.execute_query` will have their DAG fail with this change. I dont have a solution though (besides creating a new operator which I really dont like). Or maybe creating a new major version is fine? It has been a while we have not cut a new major release of the Amazon provider package. @o-nikolas @ferruzzi ?

Reply:
Yeah, I wouldn't be opposed to a major version bump, it has been over a year without a major version bump. Do we have a wishlist of other things we'd like to change as well in this bump? It's an opportunity to do some refactoring and drop some deprecated code.

Reply:
Agree, I dont think there is but we should take advantage of this opportunity